frontend/packages/console-app/src/components/guided-tours/GuidedTourCatalog.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Gallery, GalleryItem } from '@patternfly/react-core';
 import { EmptyBox } from '@console/internal/components/utils';
 import { GuidedTourCatalogItem } from './utils/guided-tour-typings';
-import GuidedTourItem from './GuidedTourItem';
+import ConnectedGuidedTourItem from './GuidedTourItem';
```

Comment:
We should keep the name the same as the component file.
As a consumer of a component we don't actually care if it is connected to redux or not.

Reply:
done